### PR TITLE
Slightly darken the color of aborted pipeline stages

### PR DESCRIPTION
**Pull request by fabianofranz**
Makes it concise with the chart label and improves contrast. @spadgett

### CODE REVIEW

File: dist/styles/main.css [hunk 1]
```diff
@@ -4098,7 +4098,7 @@ to{background-color:transparent}
 }
 @keyframes fadeInProgress{to{background-color:#0088ce}
 }
-@keyframes fadeInAborted{to{background-color:#e8e8e8}
+@keyframes fadeInAborted{to{background-color:#d6d6d6}
```
Comment (spadgett) [inline dist/styles/main.css]: Should we use `d1d1d1` from the Patternfly palette?

https://www.patternfly.org/styles/color-palette/#_

**Comment by spadgett** [comment]
@fabianofranz mind posting a screenshot?

**Comment by fabianofranz** [comment]
@spadgett

**Comment by fabianofranz** [comment]
@spadgett `#d1d1d1` looks good. Screenshot:

aborted

**Comment by spadgett** [comment]
[merge]

**Pull request merged**